frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/PipelineRuns.tsx

```diff
@@ -10,39 +13,44 @@ import { ListFilterId, ListFilterLabels } from '../../../utils/pipeline-utils';
 import PipelineRunsList from '../../pipelineruns/list-page/PipelineRunList';
 import { PipelineDetailsTabProps } from './types';
 
-export const runFilters = [
-  {
-    filterGroupName: 'Status',
-    type: 'pipelinerun-status',
-    selected: [
-      ListFilterId.Succeeded,
-      ListFilterId.Running,
-      ListFilterId.Failed,
-      ListFilterId.Cancelled,
-    ],
-    reducer: pipelineRunFilterReducer,
-    items: [
-      { id: ListFilterId.Succeeded, title: ListFilterLabels[ListFilterId.Succeeded] },
-      { id: ListFilterId.Running, title: ListFilterLabels[ListFilterId.Running] },
-      { id: ListFilterId.Failed, title: ListFilterLabels[ListFilterId.Failed] },
-      { id: ListFilterId.Cancelled, title: ListFilterLabels[ListFilterId.Cancelled] },
-    ],
-    filter: pipelineRunStatusFilter,
-  },
-];
+export const runFilters = (t: TFunction): RowFilter[] => {
+  return [
+    {
+      filterGroupName: t('pipelines-plugin~Status'),
+      type: 'pipelinerun-status',
+      selected: [
+        ListFilterId.Succeeded,
+        ListFilterId.Running,
+        ListFilterId.Failed,
+        ListFilterId.Cancelled,
+      ],
```

Comment:
Looks like this `selected` prop doesn't exist. Thanks to TS for that finding. :)

The `PipelineRun` list shows all items when nothing is selected (as most other lists). So let us drop this completely.

```suggestion
```

Reply:
Updated the PR.